### PR TITLE
docs: ngComponentOutlet doc updated with new Injector creation

### PR DESCRIPTION
**Pull request by kwiateusz**
## PR Type
What kind of change does this PR introduce?

- [x ] Documentation content changes

## Other information

In docs is used depracated method (since v5) of creating Injector so I've updated it.

### CODE REVIEW

**Comment by kwiateusz** [comment]
I signed it!

File: packages/examples/common/ngComponentOutlet/ts/module.ts [hunk 1]
```diff
@@ -56,7 +56,10 @@ class NgTemplateOutletCompleteExample {
   myContent = [[document.createTextNode('Ahoj')], [document.createTextNode('Svet')]];
 
   constructor(injector: Injector) {
-    this.myInjector = ReflectiveInjector.resolveAndCreate([Greeter], injector);
+    this.myInjector = Injector.create({
+      providers: [Greeter],
```
Comment (brandonroberts) [inline packages/examples/common/ngComponentOutlet/ts/module.ts]: To use the Static Injector, `Greeter` must be defined using a provider object

Comment (brandonroberts) [reply packages/examples/common/ngComponentOutlet/ts/module.ts]: The code also needs to be formatted. Run `gulp format:changed` after your changes are done to format the changed files using clang.

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Comment by kwiateusz** [comment]
@brandonroberts I've made changes but test still fails and in logs I don't see that is caused by me.

**Comment by brandonroberts** [comment]
@kwiateusz The provider needs the `deps` array. 

```
{ provide: Greeter, deps: [] }
```

**Review by brandonroberts — APPROVED** [review]

**Pull request closed without merging**